packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,143 +24,61 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-button-border-radius;
```

Comment:
Can you use a radius token or spacing here and not button tokens for chips?

Reply:
Yes. I was just about to ask about the border-radius of Chip. I checked the documentation and found that there is no proper border-radius (0.5rem). Is it better to add a token of 0.5rem than to use spacing? Which do you think is better?thx

Reply:
I would do `$bpk-border-radius-xs * 2`